src/dotnet/Users.Contracts/UserConstants.cs

```diff
@@ -10,4 +10,9 @@ public static class Admin
 #pragma warning disable CA2211
         public static Session Session = null!;
     }
+
+    public static class Claims
+    {
+        public static readonly string Status = "urn:actual.chat:status";
```

Comment:
Prefer properties over fields for public APIs.

Reply:
done